gubernator/main.py

```diff
@@ -34,11 +34,8 @@
 import filters
 
 BUCKET_WHITELIST = {
-    'kubernetes-jenkins',
-    'kube_azure_log',
-    'rktnetes-jenkins',
-    'kubernetes-github-redhat',
-    'kopeio-kubernetes-e2e',
+    re.match(r'gs://([^/]+)', path).group(1)
```

Comment:
it'd also let us be a bit less opinionated and support things like s3 if we wanted.